web/js/waste.js

```diff
@@ -125,6 +125,35 @@ $(function() {
         }
     }
 
+    function display_band_pricing() {
+        var band_pricing = $('#band_pricing_data');
+
+        if (!band_pricing) {
+            return;
+        } else {
```

Comment:
Don't need all this in an else, if you've returned above, can all outdent one iyswim